integration/integration_test.go

```diff
@@ -394,12 +395,12 @@ func TestIntegrationGnbsim(t *testing.T) {
 	appUplinkBytes := metrics["app_uplink_bytes"]
 	appDownlinkBytes := metrics["app_downlink_bytes"]
 
-	if appUplinkBytes != 9000 {
-		t.Fatalf("expected app_uplink_bytes to be 9000, but got %v", appUplinkBytes)
+	if appUplinkBytes < 9000 {
+		t.Fatalf("expected app_uplink_bytes to be at least 9000, but got %v", appUplinkBytes)
 	}
 
-	if appDownlinkBytes != 9000 {
-		t.Fatalf("expected app_downlink_bytes to be 9000, but got %v", appDownlinkBytes)
+	if appDownlinkBytes < 9000 {
```

Comment:
I liked the hard equality as it ensured our packet processing and metric capturing is precisely as we expect.